composer.json

```diff
@@ -10,13 +10,14 @@
         }
     ],
     "require": {
-        "consolidation/robo": "^2",
+        "consolidation/robo": "^1.4 || ^2",
         "squizlabs/php_codesniffer": "^3.5",
         "phpstan/phpstan": "^0.12.77",
         "phpstan/phpstan-deprecation-rules": "^0.12.6",
-        "drush/drush": "^10.4",
         "async-aws/s3": "^1.8",
-        "php": ">=7.4"
+        "php": ">=7.4",
+        "drush/drush": "^8 || ^10",
+        "webflo/drupal-finder": "^1.2"
```

Comment:
I assume newer versions of drupal or symphony add this for us?

Reply:
No. This was just missing from our dependencies. It was missed because it is often already included on projects.